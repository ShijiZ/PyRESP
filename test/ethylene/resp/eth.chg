All values are reported in atomic units
%FLAG TITLE
 molecule 1: ethylene

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00   1.2441670E+00
   2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
   3   0.0000000E+00   1.7277940E+00   2.3159880E+00
   4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
   5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
   6   0.0000000E+00  -1.7277940E+00   2.3159880E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          6          0    -3.3877933E-01
   2          6          1    -3.3877933E-01
   3          1          0     1.6983356E-01
   4          1          3     1.6983356E-01
   5          1          3     1.6983356E-01
   6          1          3     1.6983356E-01

