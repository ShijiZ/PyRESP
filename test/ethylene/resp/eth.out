
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for ethylene
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: ethylene
 Total charge (ich):-99
 Number of centers:  6
    1    6    0
    2    6    1
    3    1    0
    4    1    3
    5    1    3
    6    1    3

 -----------------------------
 Atom Ivary Information
 -----------------------------
    6    0
    6    1
    1    0
    1    3
    1    3
    1    3

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =    6
 Weight factor on initial charge restraints=  0.000500
 There are  0 charge constraints

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =     6
 total number of esp points =   452

 center       X               Y               Z
    1   0.0000000E+00   0.0000000E+00   1.2441670E+00
    2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
    3   0.0000000E+00   1.7277940E+00   2.3159880E+00
    4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
    5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
    6   0.0000000E+00  -1.7277940E+00   2.3159880E+00


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 values will be set to 0

 Number of unique UNfrozen charge centers =     2

 Non-linear optimization requested.
  qchnge =    9.8962852958E-02
  qchnge =    1.0636124964E-03
  qchnge =    1.6320243334E-05
  qchnge =    2.5280470647E-07

 Convergence in    3 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: ethylene

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6        0.000000     -0.338779      0       0.001416
    2         6        0.000000     -0.338779      1       0.001416
    3         1        0.000000      0.169834      0       0.000000
    4         1        0.000000      0.169834      3       0.000000
    5         1        0.000000      0.169834      3       0.000000
    6         1        0.000000      0.169834      3       0.000000

 Sum over the calculated charges:      0.002

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          2.0365103E-02
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   5.1556727E-04
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.0680054E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.5911069E-01

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        0.00000    0.00000    0.00000    0.00000

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   -2.07271
        Y    0.00000    0.65504
        Z    0.00000    0.00000    1.41766

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X    1.41766
        Y    0.00000    0.65504
        Z    0.00000    0.00000   -2.07271